src/Tribe/Views/V2/Template/Event.php

```diff
@@ -54,8 +55,10 @@ public function __construct( Template_Bootstrap $template_bootstrap ) {
 	 */
 	public function get_path() {
 		$fake_view = View::make( 'reflector' );
-		$path = $fake_view->get_template()->get_template_file( 'index' );
-		return $path;
+		$path      = $fake_view->get_template()->get_template_file( 'index' );
+		$override  = V1_Event_Templates::getTemplateHierarchy( 'default-template' );
```

Comment:
We need to keep using the V2 loading structure. 
The old file needs to stay only for V1, we need to make sure we use `default-template` or some better name than   `index`.

Reply:
@bordoni This keeps the loading of V2, but for the `single` view (without changes in v2), if there's a theme/plugin customization, then it's loading that. This fix is solving the exact problem described on the ticket and it's for the single view only.

The `default-template` from v1 comes with a series of wrappers that even if we rename this for v2, it won't make a huge difference. The structure is different. But we could rename `index` to `default-template`, do you want me to do just that?

Reply:
Yeah, all old customizations are going to deprecated with 5.0, so yes, just moving from `index` to `default-template` would be a good start.